src/components/Select/Select.tsx

```diff
@@ -425,7 +424,6 @@ export const Select: React.FC<SelectProps> = ({
           }
         )}
         ref={buttonRef}
-        aria-describedby={buildDescribedBy(uniqueId)}
```

Comment:
Actually, I believe this is used for the help/error messages.  Let's see if we can conditionally apply this attribute, if/when we do have a help or error message?